Added the new term and synonyms Fixes #15188

BarbaraCzub opened the pull request.

BarbaraCzub committed:
Added the new term and synonyms Fixes #15188

ukemi commented:
There is no need to add part_of axon if it is already asserted as an axon part. Otherwise the term looks good. Should growth cone and other things be made children of this?

BarbaraCzub commented:
Thanks @ukemi, I'll remove the 'part_of axon'; and I'll add the part_of relations between the 'growth cone' and the 'distal axon' as well as the 'axon terminus' and the 'distal axon' (as the term definition implies).

BarbaraCzub committed:
Updated relations Fixes #15188

BarbaraCzub committed:
Updated relations2 Fixes #15188

BarbaraCzub commented:
Hi @ukemi. I am not sure what I am doing wrong, but although I added the subclass/relation part_of distal axon to the 'axon terminus' term (screenshot below), the code seems to suggest that I added this relation to the parent of the  'axon terminus' term, i.e. the 'neuron projection terminus'... which is not right, but I do not seem to be able to fix it.
![screen shot 2018-02-19 at 15 48 44](https://user-images.githubusercontent.com/16541096/36386247-07dcef5e-158d-11e8-8ab6-e58937e5ed56.png)
